packages/vow/src/E.js

```diff
@@ -186,19 +186,17 @@ const resolve = x => HandledPromise.resolve(x);
 
 /**
  * @template [A={}]
- * @template {(x: any) => Promise<any>} [U=(x: any) => Promise<any>]
  * @param {HandledPromiseConstructor} HandledPromise
  * @param {object} [powers]
- * @param {U} [powers.unwrap]
+ * @param {<T>(x: T) => Promise<EUnwrap<T>>} [powers.unwrap]
  * @param {A} [powers.additional]
  */
-const makeE = (
-  HandledPromise,
-  {
+const makeE = (HandledPromise, powers = {}) => {
```

Comment:
why does SDK have it's own `makeE`?  How does this differ from Endo's? Not in scope of this PR but very important to document.